data/json/recipes/recipe_deconstruction.json

```diff
@@ -3057,5 +3057,49 @@
     "time": "30 s",
     "qualities": [ { "id": "HAMMER", "level": 1 } ],
     "components": [ [ [ "platinum_small", 2 ] ] ]
+  },
+  {
+    "result": "razor_shaving",
+    "type": "uncraft",
+    "skill_used": "fabrication",
+    "difficulty": 0,
+    "time": "1 s",
+    "components": [ [ [ "plastic_chunk", 1 ] ], [ [ "razor_blade", 1 ] ] ]
+  },
+  {
+    "result": "hair_dryer",
+    "type": "uncraft",
+    "skill_used": "mechanics",
+    "difficulty": 1,
+    "time": "1 m",
+    "qualities": [ { "id": "SCREW", "level": 1 } ],
+    "components": [ [ [ "plastic_chunk", 2 ] ], [ [ "motor_micro", 1 ] ], [ [ "cable", 1 ] ], [ [ "wire", 1 ] ] ]
```

Comment:
maybe add heating element?

Reply:
I'll replace the Wire with a Heating Element. It's not exactly enough wire to build a shopping basket, no...